osu.Framework.iOS/GameAppDelegate.cs

```diff
@@ -30,7 +30,7 @@ public override bool FinishedLaunching(UIApplication application, NSDictionary l
             Window.MakeKeyAndVisible();
 
             // required to trigger the osuTK update loop, which is used for input handling.
-            gameView.Run();
+            gameView.RunWithFrameInterval((int)UIScreen.MainScreen.MaximumFramesPerSecond);
```

Comment:
```suggestion
            gameView.RunWithFrameInterval(0); // parameter of zero uses device refresh rate.
```

Proposing as a safe alternative without potential `osuTK` changes.